karrot/groups/serializers.py

```diff
@@ -76,8 +81,17 @@ class GroupDetailSerializer(GroupBaseSerializer):
     timezone = TimezoneField()
 
     # setting constants
-    member_inactive_after_days = serializers.ReadOnlyField(default=settings.NUMBER_OF_DAYS_UNTIL_INACTIVE_IN_GROUP)
-    issue_voting_duration_days = serializers.ReadOnlyField(default=settings.VOTING_DURATION_DAYS)
+    member_inactive_after_days = serializers.SerializerMethodField()
+
+    @staticmethod
+    def get_member_inactive_after_days(_) -> int:
+        return settings.NUMBER_OF_DAYS_UNTIL_INACTIVE_IN_GROUP
+
+    issue_voting_duration_days = serializers.SerializerMethodField()
+
+    @staticmethod
+    def get_issue_voting_duration_days(_) -> int:
+        return settings.VOTING_DURATION_DAYS
```

Comment:
It's a shame there isn't a typed version of the `ReadOnlyField` to avoid these methods :/

Reply:
I think drf-spectacular is able to derive typing from `ReadOnlyField´ if the `source` parameter is given; our use case is a bit special by just providing the `default` value.